StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/SimpleMandateTextView.swift

```diff
@@ -1,16 +1,17 @@
 //
-//  CashAppMandateView.swift
+//  SimpleMandateTextView.swift
```

Comment:
FYI @porter-stripe I noticed CashAppMandateView and SepaMandateView were identical, so I combined into `SimpleMandateTextView.